examples/shaders/sky-shader/src/lib.rs

```diff
@@ -163,12 +163,12 @@ pub fn fs(screen_pos: Vec2) -> Vec4 {
     let clip_pos = screen_pos.extend(1.0).extend(1.0);
     let world_pos = {
         let p = clip_to_world.mul_vec4(clip_pos);
-        p.truncate() / p.w()
+        Vec3::from(p.truncate() / p.w())
```

Comment:
silly thing you didn't change so it's not super relevant here, but I just read glam's 0.10 changelog, and `transform_point3` now divides by `w` as of `0.9.4`, so we could change this to be `let word_pos = clip_to_world.transform_point3(screen_pos.extend(1.0));`.